test/ProblemDetailsMiddlewareTest.php

```diff
@@ -92,6 +93,7 @@ public function testThrowableRaisedByHandlerResultsInProblemDetails(string $acce
         self::assertSame($expected, $result);
     }
 
+    #[WithoutErrorHandler]
```

Comment:
This should only affect failing case when middleware does not register error handler, is that correct?

Since it would override phpunit error handler

Reply:
The test depends on an E_USER_ERROR being triggered to pass. PHPUnit's error handler swallows it, so the test fails. #[WithoutErrorHandler] disables PHPUnit's error handler just for that test.